projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -795,6 +795,13 @@ export interface FeatureTogglesInterface {
    * After providing email user will be redirected back to checkout.
    */
   opfEnablePreventingFromCheckoutWithoutEmail?: boolean;
+
+  /**
+   * When enabled, added to the default configuration
+   * a second route variant for `product`,
+   * which doesn't require the `name` parameter.
+   */
+  enableProductRouteShorterMatcher?: boolean;
```

Comment:
What about describing the change precisely with the name of the toggle, and giving the example in the JSDoc, the following way: ?

```suggestion
  /**
   * When enabled, the default routing config for the product page is no longer just:
   * `paths: ['product/:productCode/:name']`
   * but:
   * `paths: ['product/:productCode/:name', 'product/:productCode'],`
   * 
   * It means that the old URL scheme of generating links and matching URLs is preserved,
   * but now also a shorter alias (without product) name is accepted when matching the URL.
   */
  defaultProductPageRouteAllowsNoProductName?: boolean;
```